.env.example

```diff
@@ -1,34 +1,75 @@
-# Docker settings
+## Docker settings
+
+# Traefik requires this file to be able to route the requests to the containers
 DOCKER_SOCKET=/var/run/docker.sock
 
-# Domain settings
+## Domain settings
```

Comment:
If this is a higher category `domain` then the following should all be related to domain configuration and they are not. Shouldn't it be General or global? Is this even necessary?

Reply:
@m8vago thoughts?